bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/DynamicScreensSpaceErrorOptions.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components'
+import PropTypes from 'prop-types';
+import { Message, Checkbox, NumberInput } from 'oskari-ui';
+import { Numeric } from '../Numeric';
+import { Controller } from 'oskari-ui/util';
+import { StyledFormField } from './styled';
+import { JsonInput } from '../JsonInput';
+import { InfoTooltip } from '../InfoTooltip';
+
+
+const StyledDynamicScreenSpaceErrorOptionsContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+const StyledDynamicScreenSpaceErrorOption = styled.div`
+    display: flex;
+    align-items: center;
+    height: 32px;
+    margin-top: 10px;
+`;
+
+const StyledInputContainer = styled.div`
+    padding-left: 10px;
+`;
+
+const dynamicScreenSpaceErrorTemplate = `Optimization option.
+Reduce the screen space error
+for tiles that are further
+away from the camera.`;
+
+const dynamicScreenSpaceErrorDensityTemplate = `Density used to adjust
+the dynamic screen space error,
+similar to fog density.`;
+
+const dynamicScreenSpaceErrorFactorTemplate = `A factor used to increase
+the computed dynamic
+screen space error.`;
+
+const dynamicScreenSpaceErrorHeightFalloffTemplate = `A ratio of the tileset's
+height at which the density
+starts to falloff.`;
```

Comment:
These are not localized but I'm not sure if they need to be. Perhaps it's best to keep it this way.

Reply:
Add a comment here that these descriptions have been copied from CesiumJS docs: https://cesium.com/docs/cesiumjs-ref-doc/Cesium3DTileset.html